### ISSUE TITLE
there was no master to stop

### ISSUE THREAD

**Issue by shasderias** [rollcat/judo#2]
I'm getting "there was no master to stop" messages logged when running the hello.sh Hello World.

```
Running: [ahost bhost]
ahost: HELLO FROM ahost!
ahost: there was no master to stop
bhost: HELLO FROM bhost!
bhost: there was no master to stop
Success: [bhost ahost]
```

Problem appears to be due to how Proc's IsAlive() is written:

```
// IsAlive reports whether the process is still running.
func (proc Proc) IsAlive() bool {
	return proc.cmd == nil
}
```

I think the equality test should be an inequality test?

If proc.cmd == nil, the process is not alive.

**Comment by rollcat** [comment]
I can't believe this has been staring me in my face for so long. Good catch.

**Comment by rollcat** [comment]
Closed via #3, thanks.
